wallet-new/README.md

```diff
@@ -152,7 +180,8 @@ Now use following command (from the `cardano-sl` *root* directory):
 $ curl -X POST                                  \
        -H "Content-Type: application/json"      \
        -d '"PATH_TO_SECRET_KEY"'                \
-       --cacert scripts/tls-files/ca.crt
+       --cacert scripts/tls-files/ca.crt        \
```

Comment:
we probably should remove all mention of `scripts/tls-files` as none of the code uses the certs there anymore.

Reply:
Well, that's not exact unfortunately. Running the node with `stack` uses the certificates in `scripts/` by default. I'd happy to remove the default and force users to provide valid and custom certificates on each run but that's an extra overhead :confused: